### ISSUE TITLE
No Ambience in Zero Light

### ISSUE THREAD

**Issue by naqaden** [CreativeMD/AmbientSounds#32]
When I walk into the darkest part of caves (light = 0), cave ambience stops. I tested with only AmbientSounds installed.

AmbientSounds v3.0.1
Forge v14.23.5.2808
Minecraft v1.12.2

**Comment by CreativeMD** [comment]
You also need to be underground. Type in /ambient-debug to see all stats. You can also send me a screenshot of it, but I'm pretty sure the cave sound is working. You need to be below the average height by 10 blocks.

Should I adjust the height setting? I don't want the cave sound to kick in too early.

**Comment by naqaden** [comment]
I used my test save again and loaded back in a zero-light spot with no ambience. I turned on the debug info and walked toward a distant torch until I was in light level 1, causing the cave sounds to start (screenshot 1). Then I backed away from the torch into zero-light again and the sound faded to silence (screenshot 2).

I can get the same result without moving by placing and breaking torches near me.

https://imgur.com/a/W8moR18
